### PR TITLE
Doc error-reporting integration with logs

### PR DESCRIPTION
**Pull request by ofrobots**
Document loggin-{winston,bunyan} and error-reporting from each other's
README files.

### CODE REVIEW

File: packages/logging-bunyan/README.md [hunk 1]
```diff
@@ -66,6 +66,10 @@ var loggingBunyan = require('@google-cloud/logging-bunyan')({
 // ...you're good to go!
 ```
 
+## Error Reporting
+
+Any `Error` objects you log at severity `error` at higher can automatically be picked up by [Stackdriver Error Reporting][error-reporting] if your application is running on Google Cloud Platform. You may also want to see [@google-cloud/error-reporting][@google-cloud/error-reporting] module which provides direct access to the Error Reporting API.
```
Comment (stephenplusplus) [inline packages/logging-bunyan/README.md]:
```diff
- Any `Error` objects you log at severity `error` at higher
+ Any `Error` objects you log at severity `error` or higher
```

Comment (stephenplusplus) [inline packages/logging-bunyan/README.md]:
```diff
- You may also want to see [@google-cloud/error-reporting][@google-cloud/error-reporting] module
+ You may also want to see the [@google-cloud/error-reporting][@google-cloud/error-reporting] module
```

File: packages/logging-winston/README.md [hunk 2]
```diff
@@ -70,6 +70,10 @@ winston.add(transport, {
 // ...you're good to go!
 ```
 
+## Error Reporting
+
+Any `Error` objects you log at severity `error` at higher can automatically be picked up by [Stackdriver Error Reporting][error-reporting] if your application is running on Google Cloud Platform. You may also want to see [@google-cloud/error-reporting][@google-cloud/error-reporting] module which provides direct access to the Error Reporting API.
```
Comment (stephenplusplus) [inline packages/logging-winston/README.md]:
```diff
- Any `Error` objects you log at severity `error` at higher
+ Any `Error` objects you log at severity `error` or higher
```

Comment (stephenplusplus) [inline packages/logging-winston/README.md]:
```diff
- You may also want to see [@google-cloud/error-reporting][@google-cloud/error-reporting] module
+ You may also want to see the [@google-cloud/error-reporting][@google-cloud/error-reporting] module
```

File: packages/logging-bunyan/README.md (hunk 1 above)
Comment (ofrobots) [inline packages/logging-bunyan/README.md]: done

Comment (ofrobots) [inline packages/logging-bunyan/README.md]: done

File: packages/logging-winston/README.md (hunk 2 above)
Comment (ofrobots) [inline packages/logging-winston/README.md]: done

**Pull request merged**